subsys/net/l2/ethernet/dsa/dsa_tag_netc.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_SUBSYS_DSA_TAG_NETC_H_
+#define ZEPHYR_SUBSYS_DSA_TAG_NETC_H_
+
+#ifdef CONFIG_DSA_TAG_PROTOCOL_NETC
+struct net_if *dsa_tag_netc_recv(struct net_if *iface, struct net_pkt *pkt);
+struct net_pkt *dsa_tag_netc_xmit(struct net_if *iface, struct net_pkt *pkt);
+#else
+static inline struct net_if *dsa_tag_netc_recv(struct net_if *iface, struct net_pkt *pkt)
+{
+	return iface;
+}
+
+static inline struct net_pkt *dsa_tag_netc_xmit(struct net_if *iface, struct net_pkt *pkt)
+{
+	return pkt;
+}
```

Comment:
if not defined CONFIG_DSA_TAG_PROTOCOL_NETC, it should be notag for netc switch, so no need to define these two functions in "else" branch for each tag driver, the common [tag drive](https://github.com/zephyrproject-rtos/zephyr/pull/93010/files#diff-7d67425b02144b583c1670b7083e08abad1e2fc902181d6d1b9dddcc4a274467R35) will handle notag for all DSA drivers.

Reply:
Okay.
Let me change to use conditional compile to reduce resource as possible.
Thanks.